### PR TITLE
updates to stat var explorer

### PR DESCRIPTION
**Pull request by n-h-diaz**
includes 

* adds description and source url
* adds examples to default page
* updates number formats
* removes toggle from tree widget
* some style changes

### CODE REVIEW

**Comment by pradh** [comment]
For the cursory audience, could you please include a description of the updates, esp user-visible ones?

**Comment by beets** [comment]
screenshots are also really useful for these. on a mac, [take a screenshot](https://support.apple.com/en-us/HT201361) to the clipboard, and paste it in the edit box in github.

File: static/js/tools/stat_var/info.tsx [hunk 1]
```diff
@@ -32,6 +32,29 @@ class Info extends Component {
...
+            </a>
+          </li>
+          <li>
+            <a href="#statVar=Daily_PrecipitationRate_RCP85">
+              Daily Precipitation Rate Based on RCP 8.5
+            </a>
+          </li>
+          <li>
+            <a href="#statVar=Annual_Generation_Energy_NonRenewableWaste">
+              Annual Generation of Non Renewable Waste
```
Comment (beets) [inline static/js/tools/stat_var/info.tsx:54]: hm that stat var seems problematic (e.g. the word energy is missing from the sv name)

how about one of these instead: https://autopush.datacommons.org/tools/stat-var#statVar=Annual_Consumption_Electricity
https://autopush.datacommons.org/tools/stat-var#statVar=Annual_Imports_Electricity

and maybe an economics sv:
https://autopush.datacommons.org/tools/stat-var#statVar=GiniIndex_EconomicActivity

**Review by beets — APPROVED** [review]

File: static/js/tools/stat_var/info.tsx (hunk 1 above)
Comment (n-h-diaz) [reply static/js/tools/stat_var/info.tsx]: added these

File: server/routes/api/stats.py [hunk 2]
```diff
@@ -234,3 +234,10 @@ def get_statvar_summary():
     return Response(json.dumps(result.get("statVarSummary", {})),
                     200,
                     mimetype='application/json')
+
+
+@bp.route('/api/stats/propvals/<path:prop>/<path:dcids>')
```
[the remark this answers is not in the document]
Comment (n-h-diaz) [reply server/routes/api/stats.py]: thanks for the explanation :) changed

**Comment by n-h-diaz** [comment]
screenshot

Screen Shot 2021-09-30 at 1 02 52 PM

**Comment by n-h-diaz** [comment]
thanks for the review!

**Review by beets — APPROVED** [review]
thx for making the udpates!

**Pull request merged**